test/ArgumentForwardingTests/ArgumentForwardingTests.cs

```diff
@@ -51,6 +51,7 @@ private void FindAndEnsureReflectorPresent()
         /// <param name="testUserArgument"></param>
         [Theory]
         [InlineData(@"""abc"" d e")]
+        [InlineData(@"""ábc"" d é")]
```

Comment:
The fact wouldn't do anything the theory doesn't, so from that perspective, there's not much value. 

The value I see in the separation is better readability of the expectations. This essentially tests that we round trip arguments properly when doing process jumps. Having the fact would just make it clear that we are testing unicode is round tripped right. 

Ultimately it's a fit/finish thing, and I don't feel too strongly that it needs to move.